docs/user-guide/common-operations.md

```diff
@@ -1,9 +1,40 @@
-# Common Operations
+# Local Common Operations
 
-### Restarting the backend
+When running `./kubehound`, it will execute the 3 following action:
+
+* run the `backend` (graphdb, datadb and UI)
```

Comment:
Is it `datadb` (which is weirdly named since it would be datadatabase?) or `storedb`?

Reply:
storedb is what I was looking for, updating it